components/BookDetails/BookDetails.js

```diff
@@ -0,0 +1,63 @@
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { StyledDetailsPage } from "./BookDetails.style";
+import { StyledImage } from "./BookDetails.style";
+import { StyledListDetails } from "./BookDetails.style";
+import { StyledDescription } from "./BookDetails.style";
+
+export default function BookDetails({
+  imageURL,
+  title,
+  author,
+  category,
+  ISBN,
+  EAN,
+  language,
+  publication,
+  pagenumber,
+  price,
+  currencyCode,
+  description,
+}) {
+  const bookDetails = [
+    { label: "Title", value: title },
+    { label: "Author", value: author },
+    { label: "Category", value: category },
+    { label: "ISBN", value: ISBN },
+    { label: "EAN", value: EAN },
+    { label: "Language", value: language },
+    { label: "Publication", value: publication },
+    { label: "Number of pages", value: pagenumber },
+    { label: "Price", value: `${price} ${currencyCode}` },
+  ];
+
+  const router = useRouter();
+  const handleClick = () => {
+    router.push("/");
+  };
```

Comment:
instead of doing this use a next Link instead of the button. If you want the link to look like a button, you can style it that way.